tests/linearGaussian_npse_test.py

```diff
@@ -234,4 +230,4 @@ def test_npse_map():
 
     map_ = posterior.map(show_progress_bars=True)
 
-    assert torch.allclose(map_, gt_posterior.mean, atol=0.2), "MAP is not close to GT."
+    assert torch.allclose(map_, gt_posterior.mean, atol=0.4), "MAP is not close to GT."
```

Comment:
this increase in tolerance is still reasonable?

Reply:
I am not sure. I ran this test in a notebook to see why it was failing with `atol=0.2`, and saw that the MAP of the approximate posterior was estimated correctly (by looking at it on the pairplot of samples from the approximate posterior), but the posterior itself we estimated was a bit off - I guess not enough to fail our posterior tests, but enough to shift the MAP enough for this test to fail with `atol=0.2`.

Reply:
got it, then it's fine I believe.